migrations/versions/0393_add_go_live_template.py

```diff
@@ -0,0 +1,101 @@
+"""
+
+Revision ID: 0393_add_service_go_live_template
+Revises: 0392_go_live_cols_non_nullable
+Create Date: 2022-12-28 12:12:12.929661
+
+"""
+import textwrap
+
+from alembic import op
+from flask import current_app
+
+revision = "0393_add_go_live_template"
+down_revision = "0392_go_live_cols_non_nullable"
+
+
+template_id = current_app.config["ORGANISATION_HAS_NEW_GO_LIVE_REQUEST_TEMPLATE_ID"]
+template_content = textwrap.dedent(
+    """\
+    Hi ((name))
+
+    ((requester_name)) has requested for ‘((service_name))’ to be made live.
+
+    # To approve or reject this request
+
+    Review this request at: ((make_service_live_link))
+
+    # If you have any questions
+
+    To ask ((requester_name)) about their service reply to this email or contact them directly at ((requester_email_address))
+
+    ***
+
+    You are receiving this email because you are a team member of ((organisation_name)) on GOV.UK Notify.
+
+    If you need help with this request or anything else, get in touch via our support page at ((support_page_link))
+
+    Thanks,
+    GOV.​UK Notify team
+
+    https://www.gov.uk/notify
+    """
+)
+
+
+def upgrade():
+    for table_name in ("templates", "templates_history"):
+        op.execute(
+            f"""
+            INSERT INTO {table_name} (
+                id,
+                name,
+                template_type,
+                created_at,
+                subject,
+                content,
+                archived,
+                service_id,
+                created_by_id,
+                version,
+                process_type,
+                hidden
+            )
+            VALUES (
+                '{template_id}',
+                'Service wants to go live (for organisation users)',
+                'email',
+                current_timestamp,
+                'Request to go live: ((service_name))',
+                '{template_content}',
+                false,
+                '{current_app.config["NOTIFY_SERVICE_ID"]}',
+                '{current_app.config["NOTIFY_USER_ID"]}',
+                1,
+                'normal',
+                false
+            )
+            """
+        )
+
+    op.execute(
+        f"""
+        INSERT INTO template_redacted
+        (
+            template_id,
+            redact_personalisation,
+            updated_at,
+            updated_by_id
+        ) VALUES (
+            '{template_id}',
+            false,
+            current_timestamp,
+            '{current_app.config["NOTIFY_USER_ID"]}'
+        )
+        """
+    )
+
+
+def downgrade():
+    for table in ("notifications", "notification_history", "template_redacted", "templates", "templates_history"):
+        op.execute(f"DELETE FROM {table} WHERE template_id = '{template_id}'")
```

Comment:
This downgrade doesn't work for the `templates` and `templates_history` table where the column is just called `id` and not `template_id`.

Reply:
Good spot, fixed in [10e2576](https://github.com/alphagov/notifications-api/pull/3682/commits/10e2576faca9df3c04bede87b1e44f15e70c4f11)